app/src/main/java/com/aacademy/homework/data/api/JsonLoad.kt

```diff
@@ -0,0 +1,128 @@
+package com.aacademy.homework.data.api
+
+import android.content.Context
+import com.aacademy.homework.data.model.Actor
+import com.aacademy.homework.data.model.Genre
+import com.aacademy.homework.data.model.MovieDetail
+import com.aacademy.homework.data.model.MovieDetailWithActors
+import com.aacademy.homework.data.model.MoviePreview
+import com.aacademy.homework.data.model.MoviePreviewWithGenres
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+
+private val jsonFormat = Json { ignoreUnknownKeys = true }
+
+@Serializable
+private class JsonGenre(val id: Int, val name: String)
+
+@Serializable
+private class JsonActor(
+    val id: Int,
+    val name: String,
+    @SerialName("profile_path")
+    val profilePicture: String
+)
```

Comment:
what the purpose of using 2 data classes for the same data and map them? ©nikslushkin